src/libs/Clipboard/index.js

```diff
@@ -0,0 +1,5 @@
+// on Web/desktop this import will be replaced with `react-native-web` so we have to import the named export;
+// according to the `react-native-web`;
```

Comment:
`@react-native-community/clipboard` will be replaced with `react-native-web` on web platform but the exports are different.
`import Clipboard from '@react-native-community/clipboard'`
`import {Clipboard} from 'react-native-web'`.
So to get the Clipboard we have to manage these imports.

`import {Clipboard} from '@react-native-community/clipboard'` on `index.js`. Here Clipboard is `undefined` until it is replaced with `react-native-web`.

Reply:
any thoughts?

Reply:
Ah hmm. Just import it directly from `react-native-web` in this case then? No need to alias then since it doesn't do what we'd expect anyway. One version is a named export and the other a default export.

Reply:
Yeah Sure.